### ISSUE TITLE
The pie chart cannot display the details of Other correctly

### ISSUE THREAD

**Issue by LampStudy** [metabase/metabase#41113]
### Describe the bug

![image](https://github.com/metabase/metabase/assets/7403734/39cd5871-1a73-48a3-8f25-37ad2a1039c1)
After updating to 0.49.3，The pie chart cannot display the details of Other correctly

### To Reproduce

The Other of each pie chart is the same

### Information about your Metabase installation

```JSON
version: 0.49.3
```

### Severity

blocking an upgrade

**Comment by jsantana3c** [comment]
I'm having the same problem.

```

{
  "browser-info": {
    "language": "it-IT",
    "platform": "Win32",
    "userAgent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/124.0.0.0 Safari/537.36",
    "vendor": "Google Inc."
  },
  "system-info": {
    "file.encoding": "UTF-8",
    "java.runtime.name": "OpenJDK Runtime Environment",
    "java.runtime.version": "11.0.23+9",
    "java.vendor": "Eclipse Adoptium",
    "java.vendor.url": "https://adoptium.net/",
    "java.version": "11.0.23",
    "java.vm.name": "OpenJDK 64-Bit Server VM",
    "java.vm.version": "11.0.23+9",
    "os.name": "Linux",
    "os.version": "5.15.146.1-microsoft-standard-WSL2",
    "user.language": "en",
    "user.timezone": "GMT"
  },
  "metabase-info": {
    "databases": [
      "oracle"
    ],
    "hosting-env": "unknown",
    "application-database": "postgres",
    "application-database-details": {
      "database": {
        "name": "PostgreSQL",
        "version": "16.2 (Debian 16.2-1.pgdg120+2)"
      },
      "jdbc-driver": {
        "name": "PostgreSQL JDBC Driver",
        "version": "42.7.2"
      }
    },
    "run-mode": "prod",
    "version": {
      "date": "2024-05-02",
      "tag": "v0.49.8",
      "hash": "38cb850"
    },
    "settings": {
      "report-timezone": "Europe/Rome"
    }
  }
}
```

**Comment by alxnddr** [comment]
Duplicate of https://github.com/metabase/metabase/issues/42458
Fixed in 0.49.10
